subsys/net/lib/cloud/cloud.rst

```diff
@@ -0,0 +1,30 @@
+.. _cloud_api:
+
+Cloud API
```

Comment:
Is this API used in any sample?

Reply:
The API is used in the Asset tracker application

Reply:
This information is definitely worth adding in the RST file.